static/js/components/ShiftSummary.jsx

```diff
@@ -1,26 +1,26 @@
 import React, { useState } from "react";
-import { useSelector, useDispatch } from "react-redux";
+import { useDispatch, useSelector } from "react-redux";
 import makeStyles from "@mui/styles/makeStyles";
 import dayjs from "dayjs";
 // eslint-disable-next-line import/no-unresolved
 import Form from "@rjsf/mui";
 import validator from "@rjsf/validator-ajv8";
 import {
-  Paper,
   Collapse,
   Divider,
-  Tooltip,
   List,
   ListItem,
+  Paper,
+  Tooltip,
 } from "@mui/material";
 import { showNotification } from "baselayer/components/Notifications";
 import {
-  HelpOutlineOutlined,
-  ExpandMore,
   ExpandLess,
+  ExpandMore,
```

Comment:
Same comment as before, please split in 4 distinct imports for webpack splitting

Reply:
✅ Done.